### DISCUSSION TITLE
Tesla k40, 1080ti, 3060 12gb, decisions decisions..

### DISCUSSION THREAD

**Discussion by bartowski1182** [ggml-org/llama.cpp#1542] [Q&A] [answered]
Anyone have any potential insight on how these cards compare? I can snag a k40 for ~80$, but not sure if it'll be drastically slower since it's so old, but most people say it's all about the VRAM and less about the compute... So wondering if someone has tried any of these comparisons and might be able to share their experiences

**Comment by BetaDoggo** [comment, marked as answer]
I think the 3060 12GB is the easy choice for machine learning tasks. The k40 is too slow and power hungry to be worth it even at that price. It's also deprecated in cuda so it will likely not work for many projects in the near future. Compared to the 1080ti the 3060 has architectural benefits that aren't shown in on paper specs. Optimizations like xformers are more effective on 30 series cards.
